resources/lib/vrtplayer/metadatacollector.py

```diff
@@ -50,6 +52,22 @@ def get_plot(soup):
         return description
 
     @staticmethod
+    def get_broadcast_date(soup):
+        broadcast_date_tag = soup.find(class_="content__broadcastdate")
+        
+        if broadcast_date_tag is None:
```

Comment:
Could you rewrite this so the method has only one return statement. I do not like multiple return statements in methods.   When debugging i often set a breakpoint on the bottom return statement and then it would not hit. Thanks